src/blogs/performance.md

```diff
@@ -0,0 +1,72 @@
+---
+
+title: Monitoring DreamHack - the World's Largest Digital Festival
+author: rahul
+author_info: Software Engineer at MayaData, working on Cloud Native Tech
+tags: devops
+image: img1.png
+avatar: captain.png
+date: 21-03-2021
+content: Rahul Svensson (DreamHack Network Team)
```

Comment:
What is the need for this metadata ? Can't we somehow clip the content by 2-3 lines ?

Reply:
SHould be removed, kept for testing purose.